src/picomsg/src/oper_db_writer.h

```diff
@@ -3,17 +3,21 @@
 
 #include <stdint.h>
 #include "terminator_conn/c_api/conn.h"
-
+#include "oper_data_struct.h"
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 bool oper_db_writer_terminator_state_change(const uint32_t terminator_id, const enum pm_terminator_conn_state terminator_state);
 
 bool oper_db_writer_set_terminator_tunnel_counters(const uint32_t terminator_id, const uint64_t count);
-bool oper_db_writer_set_terminator_vrf_tunnel_counter(const uint32_t terminator_id, const char *tenant_id, 
+bool oper_db_writer_set_terminator_vrf_tunnel_counter(const uint32_t terminator_id, const char *tenant_id,
                                                 const char *account_id, const char *vrf_name, const uint64_t count);
-
+bool oper_db_writer_del_terminator_vrf_tunnel_counter(const uint32_t terminator_id, const char *tenant_id,
+                                                const char *account_id, const char *vrf_name);
+bool oper_db_writer_set_tunnel_oper_data(struct oper_data oper_data, bool write_uptime);
```

Comment:
Also rename to `... set_terminator_tunnel ...` (as ditto for other writes), as the term "tunnel" has a different meaning